### DISCUSSION TITLE
How long for getting a feedback from Amazon for a public listed app submission?

### DISCUSSION THREAD

**Discussion by phstc** [amzn/selling-partner-api-models#3720] [General] [answered]
I've submitted an app for a public listed, and the case got stuck with "Pending Amazon Action".

I'm about to run out of authorizations. I would love to get some insights into the process. Is there any way we can get some attention to the submission?

Is there a premium (even if paid) support for app developers?

**Comment by leonardonakagawa** [comment, marked as answer]
Hello @phstc,

Thank you for reaching out regarding App listing.

After you submit an app listing, will be created a case and the Selling Partner Appstore team will use the case log to contact you if your application requires clarification. Otherwise, The Selling Partner Appstore team will publish your app in the Selling Partner Appstore in 3 to 4 weeks.

Today Amazon does not provide a paid consultancy for app developers.

You can find details regarding this in our [List your app on the Selling Partner Appstore](https://developer-docs.amazon.com/sp-api/docs/list-your-app-on-the-selling-partner-appstore). If the information available in the documentation link provided above does not fully resolve your inquiry, please [open a support case](https://developer.amazonservices.com/support) with us.

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API Developer Support
  

https://developer-docs.amazon.com/sp-api/docs/list-your-app-on-the-selling-partner-appstore
